odoo/tests/common.py

```diff
@@ -2203,6 +2203,15 @@ def browser_js(self, url_path, code, ready='', login=None, timeout=60, cookies=N
             # code = ""
             self.assertTrue(browser._wait_ready(ready), 'The ready "%s" code was always falsy' % ready)
 
+            # slow down the network to simulate a slow connection, do it after ready code the page
+            # should not be loaded in slow mode
+            if kw.get('slow_network'):
```

Comment:
Maybe we could activate it with an environment variable like for cpu throttling.
On the other hand, I'm afraid that it could introduce more nondeterminism during runbot tests...